tests/js/spec/views/projectDetail/index.spec.tsx

```diff
@@ -48,7 +53,9 @@ describe('ProjectDetail', function () {
       ProjectsStore.loadInitialData(projects);
 
       mountWithTheme(
-        <ProjectDetails organization={organization} {...router} params={params} />,
+        <OrganizationContext.Provider value={organization}>
```

Comment:
we can pass organization in the object together with context 😉

Reply:
Hmm, I'm torn here. I would argue that using OrganizationContext.Provider mimics the real usage better than passing organization param to mountWithTheme + we have the added value of not coupling mountWithTheme with the context value

Reply:
but the `mountWithTheme` function will also wrap `OrganizationContext.Provider` with the organization you pass... the idea was to make it reusable and less verbose. The value would not be inside of the context, but rather like 

```
{
context,
organization
}
```

Reply:
Ok, that's interesting, because to me, reading `mountWithTheme(<Component/>, {context, organization etc})` poses the question of how is the context actually passed? Maybe it's just the naming of the 2nd parameter that is misleading, because it already includes a `context` key, so what is an organization in this case? 

For me, seeing OrganizationContext.Provider in the test makes it clear that we are passing the data via OrganizationContext.Provider (I agree that it's more verbose). If you look at the makeAllTheProviders component, all it does is wrap the components into Cache, Theme and OrganizationContext - this is the same as what we already do in our test + additional contexts that we do not care about in our test. I guess what I'm trying to say is that we should be cautious of unnecessary abstractions, because especially for simple cases, they do not provide any value.

Reply:
maybe we can bring this topic to the next tsc?

Reply:
Yeah, maybe. But for now, let's do what you proposed, consistency is still a nice win here